src/types.rs

```diff
@@ -316,6 +318,19 @@ impl Types {
             }
         }
     }
+
+    /// Determine if a type is an atom, that is a either an atom or a type delimited by specific
```

Comment:
This is a slightly confusing sentence.

It also feels like a bit overspecialised. More generally, you may need to know whether something has tighter priority than its surrounding or not. Not an urgent thing to fix, though.